### PR TITLE
Setting loaded configurings through netifapi

### PR DESCRIPTION
**Pull request by michalrudolf**
set up lan interface through netifapi (should be thread-safe)
func docu: https://www.nongnu.org/lwip/2_1_x/group__netifapi__netif.html#ga31755ea6dbb213236bfce19bcbe8c973

### CODE REVIEW

File: src/gui/screen_lan_settings.c [hunk 1]
```diff
@@ -82,7 +83,7 @@ static void _addrs_to_str(char *param_str, uint8_t flg) {
 
 static void _parse_MAC_addr(char *mac_addr_str) {
     uint8_t mac_addr[6] = { 0, 0, 0, 0, 0, 0 };
-    for (uint8_t i = 0; i < MAC_ADDR_SIZE; i++)
+    for (uint8_t i = 0; i < 6; i++)
```
Comment (RadekVana) [inline src/gui/screen_lan_settings.c]: i would not use "6" but sizeof(mac_addr)

File: src/gui/screen_lan_settings.c [hunk 2]
```diff
@@ -18,7 +20,6 @@
 #include <string.h>
 
 #define MAC_ADDR_START 0x1FFF781A //MM:MM:MM:SS:SS:SS
-#define MAC_ADDR_SIZE 6
```
Comment (RadekVana) [inline src/gui/screen_lan_settings.c]: why did you delete this define and used number?

File: src/gui/screen_lan_settings.c (hunk 1 above)
Comment (JustAnother1) [reply src/gui/screen_lan_settings.c]: It is not necessary to put in the size of the array if you initialize it. So this also works: uint8_t mac_addr[] = { 0, 0, 0, 0, 0, 0 };
